modules/index/src/main/java/org/apache/ignite/internal/index/IndexManagementUtils.java

```diff
@@ -259,8 +259,8 @@ static int extractIndexIdFromPartitionBuildIndexKey(String key) {
      * @param localNode Local node.
      * @param timestamp Timestamp of interest.
      */
-    static boolean isLeaseExpire(ReplicaMeta primaryReplicaMeta, ClusterNode localNode, HybridTimestamp timestamp) {
+    static boolean isPrimaryReplica(ReplicaMeta primaryReplicaMeta, ClusterNode localNode, HybridTimestamp timestamp) {
```

Comment:
The method name is now inverted (isPrimaryReplica), but the code of the method checks the opposite. It seems that it was forgotten that the logic of the method (and the logic of the call) need to be inverted too

Reply:
Fix it